examples/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ IF(WIN32)
   SET(EXAMPLES_INSTALL_BIN examples/bin/)
   SET(EXAMPLES_INSTALL_SRC examples/src/)
 ELSE()
-  SET(EXAMPLES_INSTALL_BIN ${CMAKE_INSTALL_FULL_LIBDIR}/simbody/examples/)
+  SET(EXAMPLES_INSTALL_BIN ${CMAKE_INSTALL_LIBDIR}/simbody/examples/)
```

Comment:
I now realize why this was FULL_LIBDIR: this variable is used to create a symlink, and the symlink must be given the full path. See [here](https://github.com/elen4/simbody/blob/simbodyConfig/examples/CMakeLists.txt#L157); perhaps prepend `CMAKE_INSTALL_PREFIX` here? The symlink will be broken if the user moves the installation but I think that's fine.